tests/testthat/test-Model-methods.R

```diff
@@ -7,6 +7,7 @@ test_that("doseFunction-GeneralModel returns correct dose function", {
   samples <- Samples(list(alpha0 = 1, alpha1 = 2), options = McmcOptions(samples = 1))
 
   dose_fun <- doseFunction(model, alpha0 = 1, alpha1 = 2)
+  dose_fun <- h_covr_detrace(dose_fun)
```

Comment:
A few of the tests, primarily in `tests/testthat/test-Model-methods.R` needed to be run through `h_covr_detrace` to strip out any covr injections while testing the internals.